### ISSUE TITLE
Adding a "support"-button for end-users

### ISSUE THREAD

**Issue by MrGamy** [venueless/venueless#344]
Currently it is rather complicated for end-users to contact on-platform support if they have issues with their venueless experience. 

To make it easier to contact support for end-users who don't know how to navigate venueless I suggest adding a support button which can be configured with a specific user as "support". When clicking the support button a direct message opens up where you can start messaging the designated support account.

**Comment by saschafoerster** [comment]
Yep. Usually we have an open text chat and a hotline number for support, and a team behind the scenes, if there is more than one case at the time (usually there are peak times). Chat is not really good, because every one else can see and read it, which is okayish for announcements (real announcements would be better and are in the making as far as I know) but not really good for support several people at the same time. Some people also asked for a "panic button" in BBB (or at least reachable from a room), so they need help with a presentation some one from the support can see which room needed help (for instance presentation sharing doesn't work as expected, etc). So it would be nice to see from which room the request came.
